edge/pkg/edged/edged.go

```diff
@@ -389,7 +389,16 @@ func getConfig() *Config {
 		conf.clusterDomain = ""
 	}
 
-	conf.memoryCapacity = config.CONFIG.GetConfigurationByKey("edged.edged-memory-capacity-bytes").(int)
+	//Deal with 32-bit and 64-bit compatibility issues: issue #1070
+	switch v := config.CONFIG.GetConfigurationByKey("edged.edged-memory-capacity-bytes").(type) {
+	case int:
+		conf.memoryCapacity = int64(v)
+	case int64:
+		conf.memoryCapacity = v
+	default:
+		panic("wrong type when get edged.edged-memory-capacity-bytes")
```

Comment:
we can rephrase the error to 

Invalid type for edged.edged-memory-capacity-bytes, valid types are one of [int,int64].

Reply:
@rohitsardesai83 thank you